src/containers/splashScreen/SplashScreen.css

```diff
@@ -0,0 +1,23 @@
+.splash-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    min-height: 100vh;
+}
+
+.splash-animation-container {
+    height: 50vh;
+}
+
+.splash-logo {
```

Comment:
Let's change this to `splash-title-container`

Reply:
As well as align it to center.